app/views/room/updateRoom.scala.html

```diff
@@ -75,16 +76,44 @@ <h4 style="margin-bottom: 0%">Price</h4>
                                     <input name="cost" class="phone" type="number" min="1" max="500000" required>
                                 </div>
 
-                            </div>
-                            <div class="form-group">
-                                <button class="edvin-button" id="edit-room-add-price" href="@routes.Prices.savePrice(room.id)">
+                                <button class="edvin-button pull-right" style="margin-bottom: 75px" id="edit-room-add-price" href="@routes.Prices.savePrice(room.id)">
                                     Add Price
                                 </button>
                             </div>
                         </form>
                     </div>
                     }
+                    @if(flash.containsKey("success")) {
+                        <span style="background-color: green; color: white; font-size: 20px; padding: 0px 5px 0px 5px;">
+                            @flash.get("success")
+                        </span>
+                    }
+
+                    <div>
+                        <table class="table " style="font-size: 20px; color: white">
```

Comment:
Make practices not to use inline styles, and move them to CSS files. Common practice in IT at the moment.